app/sessionStore.js

```diff
@@ -59,7 +59,14 @@ module.exports.saveAppState = (payload, isShutdown) => {
         wndPayload.tabs = wndPayload.tabs.filter((tab) => !tab.isPrivate)
       })
     } else {
-      delete payload.perWindowState
+      // Reset the window state except for the ui data
+      payload.perWindowState.forEach((wndPayload) => {
+        Object.keys(wndPayload).forEach((key) => {
+          if (key !== 'ui') {
+            delete wndPayload[key]
```

Comment:
Yes, previously it deleted everything now it preserves only the UI position params. If you selected the option to start only with home page you don't want tabs and frames from the previousl launch, no? :)